src/App.tsx

```diff
@@ -1,61 +1,98 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { FormEventHandler, useState } from 'react';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
+  const [titleError, setTitleError] = useState('');
+  const [userError, setUserError] = useState('');
+
+  const handleSubmit: FormEventHandler = (event) => {
+    event.preventDefault();
+
+    const foundUser = usersFromServer.find(user => user.name === selectedUser);
+    const newId = Math.max(...todosFromServer.map(todo => todo.id));
+
+    if (!title) {
+      setTitleError('Please enter a title');
+    }
+
+    if (!selectedUser) {
+      setUserError('Please choose a user');
+    }
```

Comment:
It would make sense to first validate and then look for data necessary to create new item. If form is invalid, prevent it from happening